unabridged-works-volume-2/original/08-swearing-under-the-gospel.adoc

```diff
@@ -258,6 +293,11 @@ which binds from swearing as a Christian can be?
 
 [.discourse-part]
 Answer.
```

Comment:
It gets funky here too, with an "answer" and a number right after it. I didn't know what to do!

Reply:
For these, by creating the new numbered group, it causes new paragraphs where there weren't paragraphs before.  So, let's remove these, and just consider them all DP's, and mark them up like this:

```ASCIIDOC
[.discourse-part]
_Answer. 1._ God (being not bound himself by the laws wherewith
```

Reply:
In fact, the more I think about it, Let's just make them `_Answer 1._` and `_Answer 2._`, etc, _removing_ the first period.  In the originals these were `Ans. 1.` and I expanded the `Ans.` to `Answer.`, but really the `.` there after `Ans` was because it was an abbreviation., and shouldn't be retained when there is a number afterward, so, let's make them all like this:

```ASCIIDOC
[.discourse-part]
_Answer 1._ God (being not bound himself by the laws wherewith
```